purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesOrchestrator.kt

```diff
@@ -1024,12 +1024,22 @@ internal class PurchasesOrchestrator constructor(
             return
         }
 
+        var previousProductId = oldProductId
+
+        if (oldProductId.contains(":")) {
```

Comment:
Nit but I'd extract this string. I bet it's already in several other places?

Reply:
Good point, extracted in https://github.com/RevenueCat/purchases-android/pull/1183/commits/1ff1a3ba8b049ddb0ec5df5d5cf13c4b62e87cf1